src/databricks/azext_databricks/custom.py

```diff
@@ -99,3 +99,70 @@ def list_databricks_workspace(cmd, client, resource_group_name=None):
     if not resource_group_name:
         return client.list_by_subscription()  # todo service 502
     return client.list_by_resource_group(resource_group_name=resource_group_name)
+
+
+def create_databricks_vnet_peering(client, resource_group_name, workspace_name, peering_name,
+                                   remote_virtual_network,
+                                   allow_virtual_network_access=None,
+                                   allow_forwarded_traffic=None,
+                                   allow_gateway_transit=None,
+                                   use_remote_gateways=None,
+                                   no_wait=False):
+    from .vendored_sdks.databricks.models._models_py3 import \
+        VirtualNetworkPeering, VirtualNetworkPeeringPropertiesFormatRemoteVirtualNetwork
+
+    peering = VirtualNetworkPeering(
+        remote_virtual_network=VirtualNetworkPeeringPropertiesFormatRemoteVirtualNetwork(id=remote_virtual_network)
+    )
+
+    if allow_virtual_network_access is None:
+        allow_virtual_network_access = True

```

Comment:
Set `default=True` in `_params.py`?

Reply:
@fengzhou-msft If I set `True` as default, then every time I execute `az databricks workspace vnet-peering update` will set `allow_virtual_network_access` to `True`, which is unexpected behavior. What I want is this default value only takes effect on `create`.